test/tests/mesh_modifiers/sidesets_between_subdomains/between_vector_blocks.i

```diff
@@ -0,0 +1,27 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+  nz = 0
+  zmin = 0
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[MeshModifiers]
+  [./subdomain_id]
+    type = AssignElementSubdomainID
+    subdomain_ids = '0 0
+                     1 2'
+  [../]
+  [./extrude]
+    type = SideSetsBetweenSubdomains
```

Comment:
`depends_on` parameter of this mesh modifier needs to be set to `subdomain_id`, the name of the other mesh modifier, because the ordering of these two modifiers is essential.